### PR TITLE
182 style filter mobile

### PR DESCRIPTION
**Pull request by edpe**
https://trello.com/c/4oecr4z8/182-style-filter-mobile-vv

### CODE REVIEW

**Review by rosejbon — APPROVED** [review]
Looks great!

File: pages/filter.tsx [hunk 1]
```diff
@@ -158,97 +158,111 @@ export const FilterPage: NextPage = () => {
...
-            })}
-          </HorizontalGroup>
-          <p>Select your age so we can tell you what you are eligible for</p>
-        </fieldset>
-      </FilterSection>
-      {/* <FilterSection>
+              </svg>
+            </h1>
+            <Link href={'/'} passHref>
+              <CloseButton>
```
Comment (Nikomus) [inline pages/filter.tsx:180]: This link will need an accessible name.

File: src/context/FilterContext.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const defaultValueShape = {
   clearAll: () => undefined,
 }
 
-export const allAges = ['<15', '15', '16', '17', '18', '18+']
+export const allAges = ['under 15', '15', '16', '17', '18', 'over 18']
```
Comment (Nikomus) [inline src/context/FilterContext.tsx:25]: The ageFilter in CardList.tsx will have to be updateded to match these

File: src/Components/Checkbox.tsx [hunk 3]
```diff
@@ -20,10 +20,32 @@ const CheckboxItem = styled.li`
   }
 
   input {
```
Comment (Nikomus) [inline src/Components/Checkbox.tsx:22]: Can you add focus styles here? You can also use the `:checked` pseudo class  selector here if that's simpler than passing props.

**Review by Nikomus — CHANGES_REQUESTED** [review]
Couple of accessibility fixes and the age filter functionality here 🏍

**Review by Nikomus — APPROVED** [review]
Ace! 📽️

**Pull request merged**